libraries/AC_Avoidance/AC_Avoid.cpp

```diff
@@ -68,6 +68,14 @@ const AP_Param::GroupInfo AC_Avoid::var_info[] = {
     // @User: Standard
     AP_GROUPINFO("BEHAVE", 5, AC_Avoid, _behavior, AP_AVOID_BEHAVE_DEFAULT),
 
+    // @Param: BK_UP_SPD
+    // @DisplayName: Back away speed
+    // @Description: Maximum speed that will be used to back away from obstacles in GPS modes (m/s)
+    // @Units: m/s
+    // @Range: 0 2
+    // @User: Standard
+    AP_GROUPINFO("BK_UP_SPD", 6, AC_Avoid, _backaway_speed, 0.5f),
```

Comment:
AVOID_BACKUP_SPD is exactly 16 characters.. that might be easier to understand than BK_UP.

Reply:
fixed